profiles/postgresql/tuned.conf

```diff
@@ -0,0 +1,54 @@
+#
+# tuned configuration for PostgreSQL servers
+#
+
+[main]
+summary=Optimize for PostgreSQL server
+include=throughput-performance
+
+[cpu]
+force_latency=1
+governor=performance
+energy_perf_bias=performance
+min_perf_pct=100
```

Comment:
Not that I mind, but these three settings are already present in `throughput-performance`. Why duplicate them?

Reply:
Dere are no reasons to   duplicate the work. The only difference is ` force_latency=1`. I will remove the others.